src/algorithms/calorimetry/CalorimeterIslandCluster.cc

```diff
@@ -80,7 +80,6 @@ static edm4hep::Vector2f globalDistEtaPhi(const CaloHit &h1, const CaloHit &h2)
 // AlgorithmInit
 //------------------------
 void CalorimeterIslandCluster::init() {
-
```

Comment:
Also extremely minor nitpick: I'd prefer leaving a space here (keeps the function name visually separated from its body and keeps it consistent with the other functions' implementation).